### ISSUE TITLE
3.1.1 - Broken in OSX - Copy / Paste File or Directory to get full path in terminal

### ISSUE THREAD

**Issue by jq-87** [vercel/hyper#5778]
- OS version and name: OSX 11.4
- Hyper.app version: 3.1.1
- Relevant information from devtools : 
```
Uncaught TypeError: this.term._core.handler is not a function
```
This feature used to work, was addressed here -> [3340](https://github.com/vercel/hyper/issues/3340)

### Issue
Steps to reproduce:

1. Copy File or Directory from finder (command + c)
2. Try pasting the file/directory into hyper (command + v) - this is when the error in console appears

What should happen after step 2:
File/Directory path SHOULD display in terminal but currently it does not anymore.

**Comment by LabhanshAgrawal** [comment]
Please test the build from #5780 and confirm if it's fixed.
https://github.com/vercel/hyper/actions/runs/1051499005

**Comment by jq-87** [comment]
Yes, It works in Hyper 3.1.0-canary.6 (stable)

Note: drag and drop of directory gives the following error in devtools:
```
Not allowed to load local resource: file:///Path/To/Folder
```

**Comment by LabhanshAgrawal** [comment]
There's a separate pr for drag and drop
You can test the build from #5781 and confirm if it's fixed.
https://github.com/vercel/hyper/actions/runs/1051719012#artifacts

**Comment by jq-87** [comment]
@LabhanshAgrawal confirmed its fixed in that build Fix file/directory drag and drop Node CI #2981

However the version from those build is: 
3.1.0-canary.6 (stable)

Shouldn't it be 3.1.1-canary.6 (stable) ?

**Comment by LabhanshAgrawal** [comment]
Ignore the version number.
